syntax = "proto3";
package com.example.grpc;

message GeneratorRequest {
  int32 first_value = 1;
  int32 last_value = 2;
}

message GeneratorResponse {
  int32 generated_value = 1;
}

service GeneratorService {
  rpc generate(GeneratorRequest) returns (stream GeneratorResponse);
}